test/variables.tf

```diff
@@ -0,0 +1,23 @@
+variable "dns_zone" {
+  default = "Z2J3KVPABDNIL1"
+}
+
+variable "domain_sfx" {
+  default = ".ops.aeternity.com"
+}
+
+variable "envid" {
+  description = "Unique test environment identifier to prevent collisions."
+}
+
+variable "vault_addr" {
+  description = "Vault server URL address"
+}
+
+variable "bootstrap_version" {
+  default = "master"
```

Comment:
Master not tag?

Reply:
Yes, I prefer to integrate modules with latest changes from its dependancies. So that the master branch of this module is up-to-date with master branches of it's dependancies.

This is working so far, I understand that we could add some trouble to make changes without first adding support for the dependancy changes.